apps/storefront/components/MdxContent/MdxContent.module.css

```
@@ -142,15 +146,15 @@
     }
   }
 
-  & *:not([data-unstyled]) {
-    & p,
-    & h2,
-    & h3,
-    & h4,
-    & ul,
-    & ol,
-    & [class^='ds-accordion'],
-    & [class^='ds-card'] {
+  & :not([data-unstyled]) {
+    &:is(p),
+    &:is(h2),
+    &:is(h3),
+    &:is(h4),
+    &:is(ul),
+    &:is(ol),
```

Just curious, any preferences on formatting here?
```suggestion
    &:is(p, h2, h3, h4, ul, ol),
```

---

not for me, this was written like this to match the selectors at the bottom, but I don't have a preference for either or

---

da foreslår jeg slå sammen siden den selectoren kan det <3